tox.ini

```diff
@@ -10,6 +10,7 @@ extras =
 deps =
     pillow8.x: Pillow<9.0.0
     pillow9.x: Pillow<10.0.0
+    pillow10.x: Pillow>=10.0.0
```

Comment:
I think we should use `<11.0.0` or `=10.0.0` otherwise when Pillow 11 will be released it will use the latest version (and probably fail)

Reply:
fixed